CesiumGltf/include/CesiumGltf/PropertyArrayView.h

```diff
@@ -58,6 +56,20 @@ template <typename ElementType> class PropertyArrayView {
         _values);
   }
 
+  bool operator==(const PropertyArrayView<ElementType>& other) const noexcept {
```

Comment:
Was this added primarily for the tests? If so, it might be better to define a free-standing `operator==` in the tests, rather than here. Or maybe even a comparison function rather than an overloaded operator? It's a little unusual to overload an operator to to element-wise comparison. `std::vector` doesn't have this operator, for instance. But if you think it's generally useful as part of the API, then you should also define `operator!=` too. It can just call `operator==` and negate it.

Reply:
I did this to enable the code here:
https://github.com/CesiumGS/cesium-native/blob/a56f7eddec540003d5ee5b92ec48a63065f55138/CesiumGltf/include/CesiumGltf/PropertyTablePropertyView.h#L272-L274

This way, it can compare a `PropertyArrayView` value to the optional `PropertyArrayView` `noData` value.

But good point, I'll define `operator!=` too.